src/fiber/channel.rs

```diff
@@ -1411,9 +1429,7 @@ where
         }
 
         if updated {
-            state
-                .generate_and_broadcast_channel_update(&self.network)
-                .await;
+            state.notify_owned_channel_updated(&self.network).await;
```

Comment:
`handle_update_command` only update our part of channel, I think we should send `OwnedChannelUpdateEvent::Updated` instead of `OwnedChannelUpdateEvent::Up` here.

Reply:
Done in https://github.com/nervosnetwork/fiber/pull/446/commits/f25f54ed9a9bda564ea9db4b1671b43826bea3f9